ballista/ui/scheduler/src/components/Summary.tsx

```diff
@@ -64,7 +64,7 @@ export const Summary: React.FunctionComponent<SummaryProps> = ({
             fontWeight={"normal"}
           >
             <Text fontWeight={"light"} mb={2}>
-              General Cluster Info
+              Cluster Details
```

Comment:
Not sure if text is necessary, but "Cluster Details" seems a bit more appropriate.